conjure-java-core/src/test/java/com/palantir/conjure/java/parameterized/TestCases.java

```diff
@@ -233,21 +233,25 @@ public final class TestCases {
                     .files(Path.of("ete-binary.yml"))
                     .options(Options.builder()
                             .requireNotNullAuthAndBodyParams(true)
+                            .jakartaPackages(true)
```

Comment:
Updating the tests to generate jakarta classes, so we can rip out the javax dependencies from our gradle